packages/cli/src/commands/organizations/list.ts

```diff
@@ -0,0 +1,40 @@
+import { vaultAPIFactory } from '@meeco/sdk';
+import { flags as _flags } from '@oclif/command';
+import { cli } from 'cli-ux';
+import { AuthConfig } from '../../configs/auth-config';
+import { OrganizationsListConfig } from '../../configs/organizations-list-config';
+import { authFlags } from '../../flags/auth-flags';
+import MeecoCommand from '../../util/meeco-command';
+
+export default class OrganizationsList extends MeecoCommand {
+  static description = 'List organization. There are three modes: validated, requested and member';
+
+  static flags = {
+    ...MeecoCommand.flags,
+    ...authFlags,
+    mode: _flags.string({
+      char: 'm',
+      default: 'validated',
+      required: false,
+      description:
```

Comment:
If there are 3 modes I think it's worth adding them as options:

```
    options: ['validated', 'requested', 'member'],          // only allow the value to be from a discrete set
```

https://oclif.io/docs/flags

Reply:
done